R/utils.R

```diff
@@ -25,20 +27,26 @@
 #' txNames <- filterTranscripts(txdb)
 #'
 filterTranscripts <- function(txdb, minFiveUTR = 30L, minCDS = 150L,
-                              minThreeUTR = 30L, stopOnEmpty = TRUE) {
+                              minThreeUTR = 30L, longestPerGene = TRUE,
+                              stopOnEmpty = TRUE) {
   txdb <- loadTxdb(txdb)
   five <- ifelse(is.null(minFiveUTR), FALSE, TRUE)
```

Comment:
seems not necesary to have this ifelse, just:
five <- !is.null(minFiveUTR)

Reply:
fixed